src/data/home.json

```
@@ -1,107 +1,106 @@
     },
     {
       "name": "gridsome-plugin-monetization",
-      "image": "logo-tool-gridsome.svg",
+      "image": "logo-plugin-gridsome.svg",
       "link": "https://github.com/Sergix/gridsome-plugin-monetization",
       "desc": "Web Monetization plugin for Gridsome",
       "version": ["1.0"]
     },
     {
       "name": "gatsby-plugin-monetization",
-      "image": "logo-tool-gatsby.svg",
+      "image": "logo-plugin-gatsby.svg",
       "link": "https://github.com/mrmuhammadali/gatsby-plugin-monetization",
       "desc": "A Gatsby plugin for react-monetize to speed up your integration with Web Monetization",
       "version": ["1.0"]
     },
     {
       "name": "ngx-monetization",
-      "image": "logo-tool-angular.svg",
+      "image": "logo-plugin-angular.svg",
       "link": "https://github.com/CDDelta/ngx-monetization",
       "desc": "Web Monetization API for Angular",
       "version": ["1.0"]
     },
     {
       "name": "Vuepress Plugin Web-Monetization",
-      "image": "logo-tool-vuepress.svg",
+      "image": "logo-plugin-vuepress.svg",
       "link": "https://github.com/spekulatius/vuepress-plugin-web-monetization",
       "desc": "Adds a Web Monetization meta tag to your VuePress website",
       "version": ["1.0"]
     },
     {
       "name": "Tessy",
-      "image": "logo-tool-tessy.svg",
+      "image": "logo-plugin-tessy.svg",
       "link": "https://github.com/VladimirMikulic/tessy",
       "desc": "Advanced Web Monetization testing library with powerful declarative API",
       "version": ["1.0"]
     },
     {
       "name": "svelte-monetization",
-      "image": "logo-tool-svelte.svg",
+      "image": "logo-plugin-svelte.svg",
       "link": "https://github.com/sorxrob/svelte-monetization",
       "desc": "A minimal and lightweight wrapper for the Web Monetization API",
       "version": ["1.0"]
     },
     {
       "name": "Moodle Webmonetization Module",
-      "image": "logo-tool-moodle-webm.svg",
+      "image": "logo-plugin-moodle-webm.svg",
       "link": "https://github.com/andrewhancox/moodle-local_webmonetization",
       "desc": "A plugin for adding a web monetization meta tag to your Moodle site",
       "version": ["1.0"]
     },
     {
       "name": "Jekyll + Webmonetization",
-      "image": "logo-tool-jekyll-webm.svg",
+      "image": "logo-plugin-jekyll-webm.svg",
       "link": "https://github.com/philnash/jekyll-web_monetization",
       "desc": "A Jekyll plugin for adding a Web Monetization meta tag to your site",
       "version": ["1.0"]
     },
     {
       "name": "Pipe Web Monetization",
-      "image": "logo-tool-pipe.svg",
+      "image": "logo-plugin-pipe.svg",
       "link": "https://www.pipewebmonetization.com/",
       "desc": "Web monetize your WordPress website and access your revenue via an analytics dashboard",
       "version": ["1.0"]
     }
   ],
   "wallets": [
+    {
+      "name": "Fynbos",
+      "image": "logo-wallet-fynbos.svg",
+      "link": "https://fynbos.app/"
```

```suggestion
      "link": "https://wallet.fynbos.app/wallet"
```

You're linking to their investment page not the wallet.

---

Fixing.